packages/pwa-buildpack/envVarDefinitions.json

```
@@ -182,6 +182,18 @@
         }
     ],
     "changes": [
+        {
+            "name": "CHECKOUT_BRAINTREE_TOKEN",
+            "type": "defaultChanged",
+            "reason": "It was unclear that developers should provide their own Braintree tokens. An example value is provided instead",
```

```suggestion
            "reason": "the confusion about whether developers should provide their own Braintree tokens for their own sites. An example value is provided instead for development purposes",
```

The "reason" property is used to complete a sentence.
The value should be a sentence fragment so that it will display the following on the CLI:
>Default value for CHECKOUT_BRAINTREE_TOKEN has changed in PWA Studio Buildpack v4.0.0, due to the confusion about whether developers should provide their own Braintree tokens for their own sites. An example value is provided instead for development purposes.

---

Fixed in https://github.com/magento/pwa-studio/pull/2008/commits/bc24b6b3182b1f33b4d39c34644acee390a231b2.